### PR TITLE
Ghost spawner naming fix

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Got merged before I could finish this bit. Lets mappers set a mob name for their ghost spawner so they don't come out named as "sleeper" or something.

### CODE REVIEW

**Comment by phil235** [comment]
mobname is used in the attack_ghost() alert message, so having it null might cause issues.

**Comment by KorPhaeron** [comment]
I'll change it to "default" then

**Comment by Anonus** [comment]
It won't runtime but yes, if it's null it'll appear as `Become ? (Warning, You can no longer be cloned!)`

**Comment by duncathan** [comment]
Why not just leave the default value as `"Unknown"`?

**Comment by KorPhaeron** [comment]
@duncathan 

Someone might actually want a mob named Unknown

**Comment by KorPhaeron** [comment]
@duncathan 

Fixed now

File: code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 /obj/effect/landmark/corpse
 	name = "Unknown"
-	var/mobname = "Unknown"  //Unused now but it'd fuck up maps to remove it now
+	var/mobname = "default"  //Use for the ghost spawner variant, so they don't come out named "sleeper"
```
Comment (Anonus) [inline code/modules/awaymissions/corpse.dm]: default it to unknown there's no good reason not to

**Pull request merged**